_specifications/specification-3-16.md

```diff
@@ -1591,6 +1591,28 @@ interface ClientCapabilities {
 		*/
 		workspaceFolders?: boolean;
 
+		/**
+		* The client has support for rename requests/notifications.
+		*
+		* Since 3.7.0

```

Comment:
Maybe this should be 3.17, right? 
Latest spec release is [3.16](https://microsoft.github.io/language-server-protocol/specifications/specification-3-16/)

Reply:
Oops, not sure where I got this. I think it should actually be v3.16 as that's still unreleased. I've updated it (and added to changelog). Thanks!